witnet.toml

```diff
@@ -29,7 +29,7 @@ genesis_path = "genesis_block.json"
 genesis_mining = false
```

Comment:
@witnet/devs should this be set to `true`? What's the use of this flag at all?

Reply:
If set to true, nodes will read the `genesis_block.json` file and try to create a genesis block. If set to false, nodes will just download the genesis block from other peers.

In order to change it to true we need to bundle the `genesis_block.json` file or otherwise the docker image won't work.

I think issue #1065 will remove this flag entirely making true the default.

Reply:
yes, I am doing that. I will upload a PR soon

Reply:
Great. Excited about #1065 !